3_Implementation/documentation/html/search/variables_7.js

```diff
@@ -0,0 +1,12 @@
+var searchData=
+[
+  ['unity_886',['Unity',['../unity_8c.html#aad738f665f16eb2336b8bc33f432d0da',1,'Unity():&#160;unity.c'],['../unity__internals_8h.html#aad738f665f16eb2336b8bc33f432d0da',1,'Unity():&#160;unity.c']]],
+  ['unitystrerr64_887',['UnityStrErr64',['../unity_8c.html#ad256fb09941c7d6fef35a9b029b85a63',1,'UnityStrErr64():&#160;unity.c'],['../unity__internals_8h.html#af5c213ecde93420aed908a92a5b32c66',1,'UnityStrErr64():&#160;unity.c']]],
+  ['unitystrerrdouble_888',['UnityStrErrDouble',['../unity_8c.html#a9cd168e94b9b20678e05a2f5ec3d377b',1,'UnityStrErrDouble():&#160;unity.c'],['../unity__internals_8h.html#ac8d03220554dfa13081f6a057ced349e',1,'UnityStrErrDouble():&#160;unity.c']]],
+  ['unitystrerrfloat_889',['UnityStrErrFloat',['../unity_8c.html#a900f212fc652672c6ef325f3099b069f',1,'UnityStrErrFloat():&#160;unity.c'],['../unity__internals_8h.html#ab644636442c612d56dfadc6970d2af67',1,'UnityStrErrFloat():&#160;unity.c']]],
+  ['unitystrerrshorthand_890',['UnityStrErrShorthand',['../unity_8c.html#ae5ca7d764811d61cac3dc0add7e68697',1,'UnityStrErrShorthand():&#160;unity.c'],['../unity__internals_8h.html#a1e2d7034b98b9532ec094f55b909e99b',1,'UnityStrErrShorthand():&#160;unity.c']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["unitystrerrshorthand_890",['UnityStrErrShorthand',['../unity_8c.html#ae5ca7d764811d61cac3dc0add7e68697',1,'UnityStrErrShorthand():&#160;unity.c'],['../unity__internals_8h.html#a1e2d7034b98b9532ec094f55b909e99b',1,'UnityStrErrShorthand():&#160;unity.c']]],
```